include/sound/sof/topology.h

```diff
@@ -75,11 +75,20 @@ struct sof_ipc_comp {
 #define SOF_MEM_CAPS_CACHE			(1 << 6) /**< cacheable */
 #define SOF_MEM_CAPS_EXEC			(1 << 7) /**< executable */
 
+#define SOF_BUF_FLAG_OVERRUN_NO_XRUN	(1 << 0) /**< overrun will cause ring
+						  * buffer overwrite, instead
+						  * of XRUN */
+#define SOF_BUF_FLAG_UNDERRUN_NO_XRUN	(1 << 1) /**< underrun will cause
```

Comment:
Ditto here.

Reply:
agree with @kv2019i these definitions are confusing as hell.

OVERRUN_NO_XRUN, so is there an overrun or not?

Reply:
`BIT(0)`, `BIT(1)`

Reply:
@lyakh For the sake of keeping the code consistent, I kept 'manual' bitshifting, as used in the whole file.
I could change that, but I strongly disagree.

Reply:
@lyakh changed to BIT(x)